routes/electionRoutes.js

```diff
@@ -1,12 +1,29 @@
 import express from 'express';
-import { startElection } from '../controllers/election.js';
-import { auth } from '../middleware/verifyToken.js';
+import { startElection, updateElectionPhase } from '../controllers/election.js';
+import { auth } from '../middleware/verifyToken.js'; 
 
 const router = express.Router();
 
-// Route for starting an election with no arguments except a token
-router.post('/start', auth, async (req, res) => {
-  startElection(res);
+// Route for starting an election
+router.post('/election/start', auth, async (req, res) => {
+  try {
+    await startElection(req, res);  // Call startElection with the request and response objects
+  } catch (err) {
```

Comment:
I think this is already handled in the controller but it's acceptable in any case.